src/Components/Aspire.Oracle.EntityFrameworkCore/ConfigurationSchema.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
Sorry to "move the cheese" on this, but I've implemented a generator that generates this file from attributes. See 

* https://github.com/dotnet/aspire/pull/1383
* https://github.com/dotnet/aspire/pull/1423

Can you update this PR to use the new generator?

Reply:
Basically add an AssemblyInfo.cs file with:

```C#
[assembly: ConfigurationSchema("Aspire:Oracle:EntityFrameworkCore", typeof(OracleEntityFrameworkCoreSettings))]

[assembly: LoggingCategories("all the log categories below")]
```

And then `dotnet build --no-incremental /p:UpdateConfigurationSchema=true .\src\Components\Aspire.Oracle.EntityFrameworkCore` to generate and update the file.

Reply:
This is great, good job!